packages/ckeditor5-package-generator/lib/templates/ckeditor5-metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"plugins": [
+		{
+			"name": "<%= name.split('/')[1] %>",
+			"className": "<%= name.split('/')[1].replace(/ /g, '') %>",
+			"description": "Adds text to the editor.",
+			"path": "src/myplugin.js",
+			"docs": "features/text-alignment.html",
+			"uiComponents": [
+				{
+					"name": "myButton",
+					"type": "Button",
+					"iconPath": "theme/icons/ckeditor.svg"
+				}
+			],
+			"htmlOutput": [
+				{
+					"elements": "p"
+				}
+			]
```

Comment:
The `htmlOutput` describes only new elements, attributes, etc that are added to the editor for the first time by a given plugin. The `<p>` element is already available in the editor, because it has been introduced by the `Paragraph` plugin - the `Paragraph` plugin contains the schema definition and the model ↔ view conversion.

The `MyPlugin` plugin does not introduce any new HTML output so the `htmlOutput` array can be skipped, as it is optional one.